assets/helpers/checkoutForm/checkoutForm.js

```diff
@@ -40,3 +40,32 @@ export function formFieldIsValid(params: formFieldIsValidParameters): boolean {
   const patternMatchError = !patternIsValid(params.value, params.pattern);
   return !emptyFieldError && !patternMatchError;
 }
+
+export function getTitle(contributionType: ContributionType): string {
+
+  switch (contributionType) {
+    case 'ANNUAL':
+      return 'Make an annual';
+    case 'MONTHLY':
+      return 'Make a monthly';
+    case 'ONE_OFF':
+    default:
+      return 'Make a single';
+  }
+}
+
+function isButtonFocused(event: FocusEvent, buttonClassName: string): boolean {
```

Comment:
What is the risk that this method returns a false positive i.e. 'another' button is clicked on with the same class name?

Reply:
Well you have to specify the className, and the two we use, `component-stripe-pop-up-button` and `component-cta-link--continue`, are very specific. So I don't think there is a risk here.